src/Blog_App_frontend/src/AddBlog.jsx

```diff
@@ -1,46 +1,63 @@
 // AddBlog.jsx
 
 import React from 'react';
-import { Formik, Form, Field, ErrorMessage } from 'formik';
+import { Formik, Form, Field } from 'formik';
 import * as Yup from 'yup';
-import { useNavigate } from 'react-router-dom'; // Import useNavigate hook
-import './AddBlog.css'; // Import custom CSS file for styling
+import { useNavigate } from 'react-router-dom'; 
+import { Form as BootstrapForm } from 'react-bootstrap'; 
+import './AddBlog.css'; 
+import { initialValues } from './constants'; 
 
 // Define the validation schema using Yup
-const AddBlogSchema = Yup.object().shape({
+const addBlogSchema = Yup.object().shape({
   title: Yup.string()
     .max(125, 'Title must not exceed 125 characters')
     .required('Title is required'),
   description: Yup.string().required('Description is required'),
 });
 
 function AddBlog({ handleAddBlog }) {
-  const navigate = useNavigate(); // Initialize useNavigate hook
+  const navigate = useNavigate(); 
 
   return (
     <div className="add-blog-container">
       <h2 className="page-title">Add Blog</h2>
       <Formik
-        initialValues={{ title: '', description: '' }}
-        validationSchema={AddBlogSchema}
+        initialValues={initialValues}
+        validationSchema={addBlogSchema}
         onSubmit={async (values, { setSubmitting, resetForm }) => {
-          await handleAddBlog(values); // Wait for handleAddBlog to complete
-          setSubmitting(false);
-          resetForm();
-          navigate('/blogs'); // Redirect to Blogs page
+          try {
+            await handleAddBlog(values);
+            resetForm();
+            navigate('/blogs'); 
+          } catch (error) {
+            console.error('Error adding blog:', error);
+          }
         }}
       >
-        {({ isSubmitting }) => (
+        {({ isSubmitting, errors, touched }) => (
           <Form className="add-blog-form">
             <div className="form-group">
               <label htmlFor="title">Title:</label>
-              <Field type="text" name="title" className="input" />
-              <ErrorMessage name="title" component="div" className="error-message" />
+              <Field 
+                type="text" 
+                name="title" 
+                className={`input ${touched.title && errors.title ? 'is-invalid' : ''}`} 
+              />
+              <BootstrapForm.Control.Feedback type="invalid">
+                {errors.title}
+              </BootstrapForm.Control.Feedback>
```

Comment:
You font have to use Field from fromik, you can use Form from bootstrap it has these functions inbuild

Reply:
ok changed it